features/tab-access-tabs.feature

```diff
@@ -0,0 +1,79 @@
+Feature: Get or set TabStops
+  In order to customize tab stops
+  As a python-docx developer
+  I need a TabStops object with read/write tab_stop properties
+
+
+  @wip
+  Scenario Outline: Get tab_stop objects
+    Given a paragraph having <value> tab_stop elements
+     Then paragraph_format.tab_stops contain tab_stop objects
+      
+    Examples:  values for tab_stop elements 
+      | value |
+      | some  |
+      
+
+  @wip
+  Scenario Outline: Count tab stop objects
+    Given a paragraph having <value> tab_stop elements
+      Then paragraph_format.tab_stops has <lenval> length
+      
+    Examples:  length values for tab_stop elements
+      | value | lenval |
+      | no    | 0      |
+      | some  | 3      |
+      
+
+  @wip
+  Scenario Outline: Iterate over tab stop objects
+    Given a paragraph having <value> tab_stop elements
+     Then I can iterate over the tab_stop elements
+      
+    Examples:  values for tab_stop elements 
+      | value |
+      | some  |
+
```

Comment:
Now that I think of it, I'm thinking these three tests can all be combined into a single scenario. Check out the test in features/doc-access-sections.feature:
```cucumber
  Scenario: Access section in section collection
     Given a section collection containing 3 sections
      Then len(sections) is 3
       And I can iterate over the sections
       And I can access a section by index
```

I'm thinking we should model after that one.

----

In general, you should never have a Scenario Outline containing only a single example. That's what a Scenario is for.